react/features/invite/functions.js

```diff
@@ -301,14 +301,6 @@ export function getInviteText({
         invite = `${invite}\n${dial}\n${moreNumbers}`;
     }
 
-    if (dialIn.sipUri) {
-        const sipText = t('info.inviteSipEndpoint', {
```

Comment:
shouldn't thetranslate be removed as well?

Reply:
no, we'll most probably end up adding it to the dial in page